account_asset_management/views/account_asset.xml

```diff
@@ -270,6 +270,41 @@
             </form>
         </field>
     </record>
+    <record model="ir.ui.view" id="account_asset_line_view_form">
+        <field name="name">account.asset.line.form</field>
+        <field name="model">account.asset.line</field>
+        <field name="arch" type="xml">
+            <form string="Asset Line">
+                <group>
```

Comment:
```suggestion
              <sheet>
                <group>
```
add all the content of the form in the <sheet> tag

Reply:
Done.